scripts/govuk_app_owners.coffee

```diff
@@ -0,0 +1,12 @@
+# This script teaches Hubot to answer questions like "Who owns Collections Publisher?"
+# and "Who owns publishing-api?"
+module.exports = (robot) ->
+  robot.hear /who owns (.*)\?/i, (res) ->
+    # Best guess of the intended application.
+    application = res.match[1].replace(' ', '-').toLowerCase()
+
+    robot.http("https://docs.publishing.service.gov.uk/apps/#{application}.json")
+      .get() (err, response, body) ->
+        if response.statusCode == 200
```

Comment:
I'm thinking that there may be a lot of cases where this bot is triggered inadvertently ("who owns that moldy tea cup next to the monitor?"), which could become annoying. This way we at least don't have false positives, but it's a bit of a trade-off.